docs/src/docs/asciidoc/guides/couchbase.adoc

```diff
@@ -0,0 +1,133 @@
+= Spring Session with Couchbase
+Mariusz Kopylec
+:toc:
+
+This guide describes how to use Spring Session backed by Couchbase.
+
+== Updating Dependencies
+Before you use Spring Session, you must ensure to update your dependencies.
+We assume you are working with a working Spring Boot web application.
+If you are using Gradle, ensure to add the following dependencies:
+
+.gradle.build
```

Comment:
Sure, I thought that if the whole project is build on gradle I will stick to one build technology.